acl/src/test/java/org/springframework/security/acls/jdbc/JdbcAclServiceTests.java

```diff
@@ -57,8 +57,8 @@ public void setUp() {
 	public void readAclByIdMissingAcl() {
 		Map<ObjectIdentity, Acl> result = new HashMap<>();
 		when(
-				lookupStrategy.readAclsById(anyListOf(ObjectIdentity.class),
```

Comment:
If we modify existing tests we run the risk of breaking other functionality later down the road. Please leave existing tests as is and create a new test that validates the new beahvior.

Reply:
Sorry, my fault, I was not clear on the change of the test case.

I've created a new issue / pull request to discuss and fix that issue:
https://github.com/spring-projects/spring-security/issues/6212

And I've have reverted the change on the test case :-)

Reply:
No problem. Thanks for reverting that and submitting the other PR.

Can you still add [some tests](https://github.com/spring-projects/spring-security/pull/6050#issuecomment-443025708) please? No need to add one for the null value (which as you pointed out cannot happen), but we should add one for the issue that we are fixing (i.e. ensure that non String types are converted).

Thanks!